pandas/core/tools/datetimes.py

```diff
@@ -942,133 +933,3 @@ def calc_with_mask(carg, mask):
         pass
 
     return None
-
```

Comment:
can you leave to_time here and show a FutureWarning (I know at least ibis is using this)

Reply:
updated per request